tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -177,6 +181,154 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, AllocateLoadComponentArraysTest )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, AllocateLoadComponentArraysTest" );
+
+	AllocateLoadComponentArraysDoAllocate = true;
```

Comment:
@JasonGlazer Like above, you don't need to do this if you use `TEST_F( EnergyPlusFixture, OutputReportTabularTest_AllocateLoadComponentArraysTest )`.